src/components/RegionSelect/RegionSelect.tsx

```
@@ -0,0 +1,86 @@
+
+export default function RegionSelect() {
+  const [selectedValue, setSelectedValue] = useState<CountryOption | null>(DEFAULT_OPTION)
+
+  const sortedOptions = countryOptions.sort((a, b) => {
+    if (a.group !== b.group) {
+      return a.group.localeCompare(b.group)
+    }
+    return a.label.localeCompare(b.label)
+  })
+
+  const handleChange = (event: any, newValue: CountryOption | null) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)